butterfly-core/src/main/java/com/paypal/butterfly/core/UpgradePath.java

```diff
@@ -96,6 +101,16 @@ private boolean upgradeVersionValidation(String targetVersion) {
         return false;
     }
 
+    /*
+     * Returns true if {@code targetVersion} is older than {@code currentVersion}
```

Comment:
I understand what you mean here, however, the parameter is called `upgradeVersion`, not `targetVersion`.
Please rename the parameter to `targetVersion`.

Also, please rename the method to `isOlderTargetVersion`.

Thanks.

Reply:
Made necessary changes.